corgi/tasks/monitoring.py

```diff
@@ -83,29 +84,41 @@ def email_failed_tasks():
         return
 
     failed_tasks_threshold = get_last_success_for_task("corgi.tasks.monitoring.email_failed_tasks")
-    failed_tasks = (
-        TaskResult.objects.filter(
-            status__exact="FAILURE",
-            date_done__gte=failed_tasks_threshold,
-        )
-        .order_by("task_name", "date_done")
-        .values_list("task_name", "task_args", "task_kwargs", "result", "traceback")
-        .using("read_only")
-    )
+    failed_tasks = TaskResult.objects.filter(
+        status__exact="FAILURE",
+        date_done__gte=failed_tasks_threshold,
+    ).using("read_only")
 
+    failed_tasks_count = failed_tasks.count()
     subject = (
-        f"Failed Corgi Celery tasks after "
-        f"{failed_tasks_threshold.date()}: {failed_tasks.count()}"
+        f"Failed Corgi Celery tasks after " f"{failed_tasks_threshold.date()}: {failed_tasks_count}"
     )
 
-    failed_tasks = "\n".join(
-        f"{task_name}: args={task_args}, kwargs={task_kwargs}\n"
-        f"result={result}\n"
-        f"{traceback}\n"
-        for (task_name, task_args, task_kwargs, result, traceback) in failed_tasks.iterator(
-            chunk_size=500
+    if failed_tasks_count < 5000:
```

Comment:
5000 still seems like a lot, will that not go over the email message size limit?